### PR TITLE
Show "range" instead of "max" in the GitHub comments.

### PR DESCRIPTION
**Pull request by nnethercote**
This is more informative, and matches better what is shown on the
comparison page.

### CODE REVIEW

File: site/src/comparison.rs [hunk 1]
```diff
@@ -294,7 +293,7 @@ impl Metric {
 ///
 /// This summary only includes changes that are significant and relevant (as determined by a change's magnitude).
 pub struct ArtifactComparisonSummary {
-    /// Relevant comparisons ordered by magnitude from largest to smallest
+    /// Relevant comparisons ordered from most negative to most positive
     relevant_comparisons: Vec<TestResultComparison>,
```
Comment (Kobzol) [inline site/src/comparison.rs:297]: The method `largest_change` is not implemented correctly after this change.

**Review by Kobzol — COMMENTED** [review]
Ranges are definitely better than a single number. I left some comments, but otherwise LGTM.

File: site/src/comparison.rs [hunk 2]
```diff
@@ -577,54 +604,39 @@ pub fn write_summary_table(
...
-        } else {
-            largest_regression
-        };
-
-        format!("{:.1}%", change * 100.0)
-    };
-
-    // max
+    // range
+    let f = |r: Option<&TestResultComparison>| r.unwrap().relative_change() * 100.0;
```
Comment (rylev) [inline site/src/comparison.rs:608]: I know we want to keep the name relatively terse, but the name `f` is very opaque...

File: site/src/comparison.rs [hunk 3]
```diff
@@ -544,6 +562,15 @@ pub fn write_summary_table(
             .unwrap_or_else(|| "-".to_string())
     }
 
+    fn render_range<F: FnOnce() -> (f64, f64)>(count: usize, calculate: F) -> String {
+        if count > 0 {
+            let (a, b) = calculate();
+            format!("[{a:.1}%, {b:.1}%]")
```
Comment (rylev) [inline site/src/comparison.rs:568]: I agree that using the same range notation as the comparison page is the right move. I do wonder if the `[a, b]` notation is right or using a Rust's range notation (`..=`) would be clearer.

Comment (nnethercote) [reply site/src/comparison.rs:586]: I think `a..=b` is one of Rust's least nice pieces of syntax, and that `[a, b]` is better.

**Review by Kobzol — APPROVED** [review]

**Pull request merged**